powersimdata/input/converter/helpers.py

```diff
@@ -0,0 +1,108 @@
+import pandas as pd
+
+
+def add_column_to_data_frame(data_frame, column_dict):
```

Comment:
Just looking through (and this is really just a stupid side-comment), there is the nice pandas version `assign` for that purpose
```python 
df = df.assign(**column_dict)
``` 
might be helpful for future implementations

Reply:
Should I get rid of the `add_column_to_data_frame` and used `assign` when needed?

Reply:
Done